src/huntsman/pocs/tests/test_states.py

```diff
@@ -88,6 +101,26 @@ def test_parking_ready(pocs):
         assert pocs.state == state
 
 
+def test_parking_dome_status(pocs):
+    '''
+    Test if the dome is closed after parking state.
+    '''
+    pocs.initialize()
+    pocs.get_ready()
+    pocs.next_state = 'parking'
+    pocs.goto_next_state()
+    assert not pocs.observatory.dome.is_open
+
+
+def test_ready_dome_status(pocs):
+    '''
+    Test if the dome is open after ready state.
+    '''
+    pocs.initialize()
```

Comment:
It would be good to make sure the dome is not already closed here.

Reply:
The aim of this test here is to make sure that the state `ready` can open the dome. So it is important to close the dome before executing the state. This is the reason I put `pocs.observatory.dome.close()` in the next line. I believe the latest method should be able to manage the situation if the dome is already closed (and keep it closed). Does this address your comment?